README.md

```diff
@@ -24,7 +24,9 @@ Download the [latest release](https://github.com/crowdsecurity/cs-cloudflare-bou
 tar xzvf cs-cloudflare-bouncer.tgz
 cd cs-cloudflare-bouncer/
 sudo ./install.sh # Use sudo ./install.sh --unattended  for automated setup
-systemctl status cs-cloudflare-bouncer
+sudo cs-cloudflare-bouncer -g <CLOUDFLARE_TOKEN1>, <CLOUDFLARE_TOKEN2> > cfg.yaml # auto-generate cloudflare config for provided space separated tokens 
```

Comment:
I think there are either missing `"` or there shouldn't be spaces ? (as is `-g a, b` is going to lead `b` to be seen as another arg from the shell pov)